### PR TITLE
Install 32-bit version of Java

### PR DESCRIPTION
**Pull request by EliasGabrielsson**
* Add support for x86 architecture for old hardware
* Install 32-bit version of Java which is recommended for OpenHAB

### CODE REVIEW

**Comment by EliasGabrielsson** [comment]
$arch variable not set

File: functions/java-jre.bash [hunk 1]
```diff
@@ -31,12 +49,35 @@ java_zulu(){
...
+    else
+      if is_x86_64; then 
+        dpkg --add-architecture i386
+        dpkg --add-architecture i686
+        cond_redirect apt update
+        cond_redirect apt -y install libc6:i386 libncurses5:i386 libstdc++6:i386
+      fi
+      echo -n "$(timestamp) [openHABian] Installing Zulu Enterprise 32-Bit OpenJDK... "
+      downloadPath=$(curl -s https://www.azul.com/downloads/zulu | grep -Eo "https://[a-zA-Z0-9./?=_-]*/zulu8[a-zA-Z0-9./?=_-]*i686.tar.gz" | head -n 1)
+      archName=i686
```
Comment (EliasGabrielsson) [inline functions/java-jre.bash]: Is used as i386 in /etc/ld.so.conf.d case

File: openhabian-setup.sh [hunk 2]
```diff
@@ -67,7 +67,7 @@ if [[ -n "$UNATTENDED" ]]; then
   bashrc_copy
   vimrc_copy
   firemotd_setup
-  java_zulu
+  java_zulu false
```
Comment (ThomDietrich) [inline openhabian-setup.sh]: "false" isn't really intuitive. how about a verbal argument like "force-64bit" or similar

Besides I wonder if it wouldn't make sense to divide the huge function into smaller ones!?

Comment (EliasGabrielsson) [reply openhabian-setup.sh]: Totally agree in both cases, period.

**Pull request closed without merging**